client/js/components/procedure/StatementSegmentsList/StatementSegmentsEdit.vue

```diff
@@ -259,10 +258,14 @@ export default {
         }
       }
 
-      return dpApi.patch(Routing.generate('api_resource_update', { resourceType: 'StatementSegment', resourceId: segment.id }), {}, payload)
-        .then(checkResponse)
+      const options = {
+        messages: {
```

Comment:
The error message (see line 273) should be passed here, too

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/2538/commits/9767bc743a4a4ee5798b8ea6fecfd71490ea68de